template/chat.html

```diff
@@ -34,6 +34,9 @@
         }
       },
 
+      /*
+        writes a message in the chat box.
+      */
       sendMessage: function(user, message) {
         if (message != '')
```

Comment:
That also depends on what an emptu string is for you. But, depending on the empty string definition, you might end up adding more and more tests (I am not defending the regex solution, just saying that thinking and checking all edge cases might not be as trivial as it may look like).